controllers/object_controls.go

```diff
@@ -2457,9 +2458,10 @@ func getRuntimeConfigFiles(c *corev1.Container, runtime string) (string, string,
 		} else if value := getContainerEnv(c, "RUNTIME_CONFIG"); value != "" {
 			topLevelConfigFile = value
 		}
+		// Users can opt-out of using drop-in files by setting RUNTIME_DROP_IN_CONFIG to ""
```

Comment:
@elezar @cdesiniotis  I think we should reconsider this. Treating the absence of a value or empty string as an input is bug-prone and generally counterintuitive.

I think it's better just to use a bool flag called `RUNTIME_DROP_IN_CONFIG_ENABLED` and have users explicitly opt-out.

Reply:
Note that the `RUNTIME_DROP_IN_CONFIG` will be used even when we switch to CRI-based config retrieval. The use of a drop-in or a fully-rendered config is separate from the config source.

My concern with adding yet another config option is that this is something that we need to maintain compatibility with.

As a follow-up question: Would using a reserved value like "disabled" for `RUNTIME_DROP_IN_CONFIG` to disable this instead fo `""` be more agreeable?

Reply:
I agree with Evan, and I think that an empty value is ok, not to read as disabled, simply that we have a default  (empty value) behaviour, and a non-empty value is then a call to modify the default, which in the case of `RUNTIME_DROP_IN_CONFIG` is to modify the way a runtime config is handled